Sustained Immunity Update (#897)
greenelab/covid19-review: content/10.diagnostics.md

It reported that IgG could be detected within two days of the onset of fever but that IgM could not be detected any earlier, a pattern they compared to findings in another disease caused by a HCoV, Middle East respiratory syndrome (MERS).
Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].

> Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036; @doi:10.1016/j.medj.2021.02.001].

Author: Generally speaking, the consensus is that yes infection does confer sustained immunity but that after the 6-8 month period the immunity does wane and there is increased risk of reinfection for some. It is not clear though yet, whether there is still protection from reinfection beyond that time point, and we really don't know what the story will be with the t-cell and b-cells pool. My guess is there must be some protection, but how protective that is to prevent reinfection, I really do not know. The group that produced the Gaebler study have kept some patients to do a 1 and 2 year post infection study, which will certainly provide answers to that. 

I hear your concerns though, inferring protection from these types of studies is not always indicative of reality, but my perception is that examining antibody responses and assessing neutralizing activity seems to be the gold standard for these folks.  

PS I highly recommend you listen to Christian Gaebler who is on an episode of the podcast TWIV, which I also recommend. He gives an overview of his study and what is to come for sustained immunity research with SARS-CoV-2

> Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036; @doi:10.1016/j.medj.2021.02.001].

Reviewer: Thank you so much for explaining @RLordan! And thank you for the rec, I just subscribed to that podcast!

That definitely makes sense! Maybe something we could mention in the conclusions for this paper in terms of future directions? That's fantastic they're geared up for a longitudinal study!

Dan et al. assessed sustained immunity using 245 blood samples from 188 COVID-19 positive patients [@doi:10.1126/science.abf4063]. 
The samples were collected at various time points between 6 and 240 days post-symptom onset, meaning some patients were assessed longitudinally. 
Of the samples, 43 were collected at least 6 months after symptom onset.
After 1 month, 98% of patients were seropositive for IgG to the spike protein, S. 
Moreover, S IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.

The findings of Dan et al. are in accordance with Sherina et al. who examined 119 samples from 88 donors who had mild to severe COVID-19 [@doi:10.1016/j.medj.2021.02.001]. 

> They observed a relatively stable level of IgG and plasma neutralizing antibodies up to 6 months post diagnosis.

Author: I got that they looked at antibodies up to 6 months, and reported a reduction of antibodies form 6-8 months. They reported the T- and B-cells from 6 to 8 months. 

They adapt that to conclude that immunity will last 6-8 months. Do we need to rephrase here if it is not clear?

> They observed a relatively stable level of IgG and plasma neutralizing antibodies up to 6 months post diagnosis.

Reviewer: That makes sense! That helps me understand why they portrayed it as 6-8!

> Significantly lower but considerable levels of anti-SARS-CoV-2 IgG antibodies were still present in 80% of samples obtained 6-8 months post symptom-onset.

Author: Haha nitpicking welcome. 

The line I wrote refers to page 284 on the fourth line, referring to samples obtained from 15 patients for the two month period post 6 month collection (6-8 months). They also suggest that they still had neutralizing activity (page 286/figure S1).

Have a look and see if you think. :)

> Significantly lower but considerable levels of anti-SARS-CoV-2 IgG antibodies were still present in 80% of samples obtained 6-8 months post symptom-onset.

Reviewer: Thank you so much @RLordan! I see your original numbers!

> A study by Gaebler et al. has shown that titers of IgM and IgG antibodies against the RBD decrease from 1.3 to 6.2 months post infection in 87 individuals [@doi:10.1038/s41586-021-03207-w]. 

Author: So one possibility is that people didn't seroconvert, then they had no rise in antibodies in the first place. However, they only included people that had seroconverted. 13 people were excluded from this study who did not seroconvert. You only see that in the supplementary submission files. 

I see what you are saying though. It is not possible to tell from figure 1 whether one or two participants did not follow that trend.

IgM seemed to show the greatest decrease of anti-RBD reactivity (53%), followed by IgG (32%), but a lower decline of IgA (15%) activity was observed. 

> It was noted that higher levels of anti-RBD IgG and anti-N total antibodies were detected in individuals that reported persistent post-acute symptoms at both study visits.

Author: I assumed it might have been earlier when discussing antibody diagnostics. Only way to confirm whether you had infection prior to receiving a vaccine. Because the vaccine induces antibodies for spike. Super cool

Moreover, plasma neutralizing activity decreased five-fold between 1.3 and 6.2 months in an assay of HIV-1 virus pseudotyped with SARS-CoV-2 S protein, and this neutralizing activity was directly correlated with IgG anti-RBD titres [@doi:10.1038/s41586-021-03207-w].

> These findings are in accordance with other studies that show that the majority of seroconverters have detectable, albeit decreasing levels of neutralizing antibodies at least 3-6 months post infection  [@10.1126/science.abd7728; @doi:10.1038/s41564-020-00813-8; @doi:10.1038/s41467-021-21444-5].

Reviewer 2: ```suggestion
These findings are in accordance with other studies that show that the majority of seroconverters have detectable, albeit decreasing levels of neutralizing antibodies at least 3-6 months post infection [@doi:10.1126/science.abd7728; @doi:10.1038/s41564-020-00813-8; @doi:10.1038/s41467-021-21444-5].
```

Author: Oops, thanks

Determining the potency of anti-RBD antibodies early in the course of an infection may be important moving forward as their neutralizing potency may be prognostic for disease severity and survival [@doi:10.1016/j.cell.2020.12.015].

> The development of memory B cells and memory T cells has also been assessed in several studies.

Author: lifespan, longevity? Although I don't think the lifespan has been fully assessed as memory B-cells can sometimes last for decades, but the abundance has been assessed a lot...

> The development of memory B cells and memory T cells has also been assessed in several studies.

Reviewer: I'm definitely fine leaving it the way you wrote it, it sounds like additional studies are needed to talk about the timecourse!

Dan et al. showed that SARS-CoV-2 S-specific memory B cell levels steadily rose over the first 120 days following symptom onset [@doi:10.1126/science.abf4063].
RBD-specific memory B cells had been detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @pmid:33443036], and this study confirms these finding and shows that these cells steadily increased over the 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].

> Gaebler et al. has shown that memory B cells specific to the RBD remain unaltered  and exhibit clonal turnover and antibody sequence evolution 6 months post infection indicative of prolonged germinal cell reactions  [@doi:10.1038/s41586-021-03207-w]. 

Reviewer 2: ```suggestion
Gaebler et al. have shown that memory B cells specific to the RBD remain unaltered and exhibit clonal turnover and antibody sequence evolution 6 months post infection indicative of prolonged germinal cell reactions  [@doi:10.1038/s41586-021-03207-w]. 
```

> Gaebler et al. have shown that memory B cells specific to the RBD remain unaltered and exhibit clonal turnover and antibody sequence evolution 6 months post infection indicative of prolonged germinal cell reactions  [@doi:10.1038/s41586-021-03207-w]. 

Author: Germinal center/centre, I also have no clue what the right spelling is haha I went with the former- they are temporary structures in the lymph nodes following immunization where B cells do their business.  
```suggestion
Gaebler et al. have shown that memory B cells specific to the RBD remain unaltered and exhibit clonal turnover and antibody sequence evolution 6 months post infection indicative of prolonged germinal center reactions [@doi:10.1038/s41586-021-03207-w]. 
```

The same study showed that antibodies expressed by these memory B cells have resistance to RBD mutations, greater somatic hypermutations, and increased potency, which the authors suggest might be evidence of continued evolution of humoral immunity [@doi:10.1038/s41586-021-03207-w].

> Indeed, Wheatley et al. showed that S-specific IgG^+^ memory B cells consistently increase over time and by 4 months they comprise approximately 0.8% of all IgG^+^ memory B cells, which may indicate cellular immune memory to even mild-moderate COVID-19 infection [@doi:10.1038/s41467-021-21444-5]. 

Reviewer: ```suggestion
Indeed, Wheatley et al. showed that S-specific IgG^+^ memory B cells consistently increase over time and by 4 months comprise approximately 0.8% of all IgG^+^ memory B cells, which may indicate cellular immune memory to even mild-to-moderate COVID-19 infection [@doi:10.1038/s41467-021-21444-5]. 
```

Dan et al. showed that N-specific memory B cells steadily increased up to 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
SARS-CoV-2 memory CD8^+^ T cells were also detected in 70% of 169 COVID-19 patients after 1 month [@doi:10.1126/science.abf4063], which is consistent with previous research [@doi:10.1016/j.cell.2020.05.015].
However, SARS-CoV-2 memory CD8^+^ T cells were slightly decreased (50%) 6 months post-symptom onset. 
In this same subset of COVID-19 patients, 93% of subjects had detectable levels of SARS-CoV-2 memory CD4^+^ T cells, of which 42% had more than 1% SARS-CoV-2-specific CD4^+^ T cells.
At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4^+^ T cells.

> Indeed, the S-specific memory CD4^+^ T cells abundance over time were similar to the overall SARS-CoV-2-specific CD4^+^ T cells [@doi:10.1126/science.abf4063].

Reviewer: ```suggestion
Indeed, the abundance of S-specific memory CD4^+^ T cells over time was similar to that of SARS-CoV-2-specific CD4^+^ T cells overall [@doi:10.1126/science.abf4063].
```
